src/bin/jcli_app/rest/v0/utxo/mod.rs

```diff
@@ -10,20 +10,24 @@ pub enum Utxo {
         addr: HostAddr,
         #[structopt(flatten)]
         debug: DebugFlag,
+        #[structopt(flatten)]
+        output_format: OutputFormat,
     },
 }
 
 impl Utxo {
     pub fn exec(self) {
-        let (addr, debug) = match self {
-            Utxo::Get { addr, debug } => (addr, debug),
-        };
+        let Utxo::Get {
+            addr,
+            debug,
+            output_format,
+        } = self;
         let url = addr.with_segments(&["v0", "utxo"]).unwrap().into_url();
         let builder = reqwest::Client::new().get(url);
         let response = RestApiSender::new(builder, &debug).send().unwrap();
         response.response().error_for_status_ref().unwrap();
-        let utxos: serde_json::Value = response.body().json().unwrap();
-        let utxos_yaml = serde_yaml::to_string(&utxos).unwrap();
-        println!("{}", utxos_yaml);
+        let status = response.body().json_value().unwrap();
+        let formatted = output_format.format(status).unwrap();
```

Comment:
This is a bit repetitive. Can `OutputFormat` accept the response body and then transform it as necessary? This could skip the JSON intermediate step.

Reply:
It could accept a `serde_json::Value`, so we don't serialize and deserialize pointlessly. I want to avoid accepting a REST body, this looks like a responsibility creep